plugins/source/aws/codegen/recipes/templates/resolver_and_mock_test/list_resources_1/README.md

```diff
@@ -0,0 +1,76 @@
+This folder contains templates for resolvers and mock-tests, for AWS apis of the form `List<Resource>s` (e.g. `eventbridge.ListArchives`)
+
+Example Resolver (`eventbridge.Archives`)
+```go
```

Comment:
I like the idea of including an example, but this is going to subject to frequent change, maybe we can just link to the eventbridge resource from here?